h/assets.yaml

```diff
@@ -203,46 +92,33 @@ app:
     - jquery
     - angular
     - angular_animate
+    - angular_bootstrap
     - angular_resource
     - angular_route
     - angular_sanitize
+    - angular_tags_input
+    - annotator
+    - autofill
+    - bind
+    - katex
+    - momentjs
+    - url
+    - helpers
     - app_css
     - app_js
-    - helpers
 
 app_js:
   output: scripts/app.min.js
-  contents:
-    - jschannel
-    - jwz
-    - katex
-    - momentjs
-    - pagedown
-    - autofill
-    - unorm
-    - url
-    - uuid
-    - angular_tags_input
-    - annotator
-    - annotator_plugins_app
-    - filters: uglifyjs
-      contents:
-        - filters: coffeescript
-          output: scripts/app.js
-          contents:
-            - h:static/scripts/app.coffee
-            - h:static/scripts/controllers.coffee
-            - h:static/scripts/directives.coffee
-            - h:static/scripts/directives/*.coffee
-            - h:static/scripts/filters.coffee
-            - h:static/scripts/searchfilters.coffee
-            - h:static/scripts/services.coffee
-            - h:static/scripts/*-service.coffee
-            - h:static/scripts/streamsearch.coffee
-            - h:static/scripts/annotation-sync.coffee
-            - h:static/scripts/annotation-ui-sync.coffee
-            - h:static/scripts/bridge.coffee
-            - h:static/scripts/discovery.coffee
+  filters:
+    - browserify
+    - uglifyjs
+  contents: h:static/scripts/app.coffee
+  depends:
+    - h:static/scripts/*.coffee
```

Comment:
Oh, that's awesome. It never occurred to me that the depends supported globbing.